### PR TITLE
Steve final ff features

### PR DESCRIPTION
**Pull request by Steven-David215**

### CODE REVIEW

**Review by TroyDexter — APPROVED** [review]
Apprved

**Review by markfilliph — APPROVED** [review]
approved

**Review by ValeriaRincon — APPROVED** [review]
Looks good

File: Assets/Scripts/DialogueSystem/DialogueManager.cs [hunk 1]
```diff
@@ -104,7 +104,7 @@ public void StartConversation(Conversation convo, ConvoFinishedCallback callb)
 
         if (convoInProgress == false)
         {
-            Debug.Log("Yup, stuff happened");
+            //Debug.Log("Yup, stuff happened");
```
Comment (DoopweeSLC) [inline Assets/Scripts/DialogueSystem/DialogueManager.cs:107]: cool, disregard prior feecback on this, great job

File: Assets/Scripts/FFSoakingSpirits/FFFire.cs [hunk 2]
```diff
@@ -81,6 +100,17 @@ void Update()
         if(fireLevel <= 2)
         {
             fireLevel++; // up to Level2 which maxLevel could be a variable maybe?
```
Comment (DoopweeSLC) [inline Assets/Scripts/FFSoakingSpirits/FFFire.cs:102]: would this not allow it to go to 3? i feel like at the end your firelevel would be bouncing between 2 and 3 going into the if then else each other frame

Comment (Steven-David215) [reply Assets/Scripts/FFSoakingSpirits/FFFire.cs:102]: good thing the function is only called if the fire is < level2. that's a whoops that was accidentally avoided.

**Pull request merged**